suitcase/hdf5.py

```diff
@@ -96,25 +96,23 @@ def export(headers, filename, stream_name=None, fields=None, timestamps=True, us
                                                 fletcher32=True)
                     data = [e['data'][key] for e in events]
                     data = np.array(data)
-                    try:
+                    if data.dtype.kind in ['S', 'U']:
```

Comment:
I tried to use the information in descriptors. It looks like 2D array of string and 2D array of number are both defined as 'array' in dtype. So I still need to use extra information like np.dtype to get the correct data type.